src/runtime-rs/crates/hypervisor/src/ch/inner_device.rs

```diff
@@ -127,6 +128,9 @@ impl CloudHypervisorInner {
     }
 
     async fn handle_hvsock_device(&mut self, _cfg: &HybridVsockConfig) -> Result<()> {
+        // uds path is to be used for adding hvsock device in VMM.
+        let _uds_path = [&self.jailer_root, DEFAULT_HYBRID_VSOCK_NAME].join("/");
```

Comment:
If the hvsock logic is in the device manager entirely, we can simply return Ok(()) here.

Reply:
fixed